service.yaml

```diff
@@ -3,28 +3,31 @@ arguments:
   ciOptions:
     skipDocker: true
     skipE2e: true
-  reportingTeam: fnd-dt
```

Comment:
The reportingTeam should not be deleted.

Reply:
I see, it was moved down.

Reply:
Yeah, rewriting the `service.yaml` via Go reorders a lot of things.